### ISSUE TITLE
CRAM file support

### ISSUE THREAD

**Issue by jjfarrell** [pezmaster31/bamtools#149]
Are there any plans to support CRAM files with bamtools?

**Comment by sfchen** [comment]
I just planned to raise the same issue :)

CRAM is wanted.

**Comment by gnarzisi** [comment]
Yes, same desire here for CRAM support. I guess that would require a name change to cramtools!
